src/test/java/com/meilisearch/sdk/TestUtils.java

```diff
@@ -0,0 +1,53 @@
+package com.meilisearch.sdk;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.lang.reflect.Type;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+public class TestUtils {
+
+	String movies_data;
+	Movie[] movies = new Movie[10];
+	Gson gson = new Gson();
+
+	public TestUtils () {
+		this.movies = moviesStringToJson(moviesFileToString());
+		return;
+	}
+
+	public String moviesFileToString () {
+		String content = "";
+		String filePath = "src/test/java/com/meilisearch/sdk/movies.json";
+		try
+		{
+			content = new String ( Files.readAllBytes( Paths.get(filePath) ) );
+		} 
+		catch (IOException e) 
+		{
+			e.printStackTrace();
+		}
+		this.movies_data = content;
+		return content;
+	}
+
+	public Movie[] moviesStringToJson (String movies_str) {
+		Movie[] movies = new Movie[10];
+		Type MOVIE_TYPE = new TypeToken<Movie[]>() {
+		}.getType();
+		movies = gson.fromJson(movies_str, MOVIE_TYPE);
+		return movies;
+	}
```

Comment:
you could replace this method with a simple 
`gson.fromJson(movies_str,TypeToken.getParameterized(List.class, Movie.class))` to get a List of Movies
or 
`gson.fromJson(movies_str,Movie[].class)` to get an Array of Movies